WalletWasabi.Fluent/Converters/ResolveApplicationResourceConverter.cs

```diff
@@ -0,0 +1,33 @@
+using System.Globalization;
+using Avalonia;
+using Avalonia.Data.Converters;
+
+namespace WalletWasabi.Fluent.Converters;
+
+// Not sure if there is a straight XAML-only way to do this
+// {StaticResource ResourceKey={Binding SomeString}} doesn't work :(
+public class ResolveApplicationResourceConverter : IValueConverter
+{
+	public static readonly ResolveApplicationResourceConverter Instance = new();
+
+	public object? Convert(object? value, Type targetType, object? parameter, CultureInfo culture)
+	{
+		if (Application.Current is not { })
```

Comment:
Why not check if `Application.Current is null`? It is more readable.

```suggestion
		if (Application.Current is null)
```